default_config.toml

```diff
@@ -0,0 +1,12 @@
+## =================================================================
```

Comment:
Why TOML? Any benefits to using this over python's built-in config language?

Reply:
Partly because PennyLane uses it, and I want to maintain consistency. Also, TOML is generally easier to hand-edit than JSON, not as overcomplicated as YAML, and better suited to structured data than INI files (the format used by Python's in-built ConfigParser library).

It also seems that the Python ecosystem is moving towards TOML, e.g., PEP518 and pyproject.toml files.

Reply:
Neat, didn't know about the PEP.